quicklogic/common/cmake/quicklogic_qlf_device.cmake

```diff
@@ -57,27 +109,25 @@ function(QUICKLOGIC_DEFINE_QLF_DEVICE)
   set_target_properties(
     ${DEVICE_TYPE}
     PROPERTIES
+    USE_ROI FALSE
+    LIMIT_GRAPH_TO_DEVICE FALSE
     TECHFILE ""
     FAMILY ${FAMILY}
-    DEVICE_MERGED_FILE ${CMAKE_CURRENT_SOURCE_DIR}/${ARCH_XML}
+    DEVICE_MERGED_FILE ${CMAKE_CURRENT_SOURCE_DIR}/${ARCH_XML_NAME}
   )
 
   # .......................................................
 
-  # RR graph patch dependencies
-  set(DEVICE_RR_PATCH_DEPS "")
-  set(DEVICE_RR_PATCH_EXTRA_ARGS "")
-
   # Define the device
   define_device(
     DEVICE ${DEVICE}
     ARCH ${ARCH}
     DEVICE_TYPE ${DEVICE_TYPE}
     PACKAGES ${PACKAGE}
     ROUTE_CHAN_WIDTH ${ROUTE_CHAN_WIDTH}
-    WIRE_EBLIF ${symbiflow-arch-defs_SOURCE_DIR}/common/wire.eblif
```

Comment:
Without `WIRE_ELBIF`, how is the lookahead generated?

Reply:
If you don't specify a "wire.eblif" explicitly the default one is used: https://github.com/antmicro/symbiflow-arch-defs/blob/712493b81351450f5244814d00369020ef976171/common/cmake/devices.cmake#L556-L560